resources/aur.rb

```diff
@@ -17,14 +17,115 @@
 # limitations under the License.
 #
 
-actions :build, :install
-
 default_action :install
 
-attribute :package_name, :name_attribute => true
-attribute :version, :default => nil
-attribute :builddir, :default => "#{Chef::Config[:file_cache_path]}/builds"
-attribute :options, :kind_of => String
-attribute :pkgbuild_src, :default => false
-attribute :patches, :kind_of => Array, :default => []
-attribute :exists, :default => false
+property :package_name, String, name_property: true
+property :version, [String, NilClass], default: lazy {
+  v = ''
+  r = ''
+  a = ''
```

Comment:
These variable names should be more descriptive, so you know what they are.

Reply:
I know you just carried these over though, which I inherited from a different maintainer

Reply:
changed to more descriptive variable names